ekncontent/ekncontent/eknc-domain.c

```diff
@@ -439,6 +468,20 @@ eknc_domain_initable_init (GInitable *initable,
               return FALSE;
         }
 
+      /* get subscriptions from extensions */
+      g_autofree gchar *extensions_path = g_build_filename ("/app", "share", "ekn", "subscriptions", NULL);
```

Comment:
hmm, I think I need to consider the case where this is ran from outside the app itself, like from EknServices.